DittoDiskUsage/src/main/java/com/example/dittodiskusage/usecase/GetDiskUsageMetrics.kt

```diff
@@ -6,12 +6,12 @@ import com.example.dittodiskusage.DiskUsageState
 import com.example.dittodiskusage.METRIC_NAME
 import com.example.dittodiskusage.ROOT_PATH
 import com.example.dittodiskusage.TOTAL_SIZE
-import com.example.dittodiskusage.TWO_GIGABYTES_IN_MEGABYTES
+import com.example.dittodiskusage.FIVE_HUNDRED_MEGABYTES_IN_BYTES
 import live.ditto.dittohealthmetrics.HealthMetric
 
 class GetDiskUsageMetrics() {
     val metricName: String = METRIC_NAME
-    var isHealthyMBSizeLimit: Int = TWO_GIGABYTES_IN_MEGABYTES
+    var unhealthySizeInBytes: Int = FIVE_HUNDRED_MEGABYTES_IN_BYTES
```

Comment:
you would want to pass this in as a parameter of `execute()`.

Reply:
This is now part of `DiskUsageState` which is already passed there

Reply:
How would a client set their own unhealthy size?

Reply:
It gets passed in from whatever they set in `DiskUsageViewModel.kt` every time the callback is triggered (which I just realized was private, fixed now)